B827EBFFFE4AD528.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "b827ebfffe4ad528",
```

Comment:
Please use all uppercase for the ID here. After updating I'll merge this.
Cheers!